readme.md

```diff
@@ -1,53 +1,158 @@
+```
+npm i @bitovi/use-simple-reducer
+```
+
+<a id="the-hook"/>
+
+## The Hook
 
+#### Use
+
+In your component, call `useSimpleReducer` with:
+
+| Field          | Type                                                     | Purpose                                                                                                                                                                                                                                                                                                                                            |
+| -------------- | -------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `initialState` | any                                                      | The initial state you want to manage.                                                                                                                                                                                                                                                                                                              |
+| `actions`      | {[key: string]: </br> (state: any, payload: any) => any} | A collection of reducer methods, each to be called with </br> the`state` as the first argument and values passed to </br> the `action` method as the second argument. If you are </br> unfamiliar with the reducer pattern you can check out </br> the [redux reducer](https://redux.js.org/tutorials/fundamentals/part-3-state-actions-reducers). |
 
 ```js
-const [state, isProcessing, actions] = useAsyncReducerstate(
-    // initial state
-    {count: 0},
-    // collection of reducer methods
-    {
-        async add(state, amountToAdd ){
-            return { ...state, count: state.count + amountToAdd };
-        },
-        async subtract(state, amountToSubtract ){
-            return { ...state, count: state.count - amountToSubtract };
-        }
-    }
-})
+function Counter() {
+  const [state, actions, queue, error] = useSimpleReducer(
+      // initial state
+      {count: 0},
+      // collection of reducer methods
+      {
+          async add(state, amountToAdd ){
+              return { ...state, count: state.count + amountToAdd };
+          },
+          async subtract(state, amountToSubtract ){
+              // Performing a GET request to fetch an offset value to be subtracted from 'amountToSubract'
+              fetch("https://counter.com/offset")
+              .then(response => response.json())
+              .then(amount => return { ...state, count: (state.count - amountToSubtract - amount.offset) })
+          }
+      }
+   )
+
+   return(
```

Comment:
This part is duplicated below.

Reply:
One of the comments in the user testing was to include the whole example at the beginning and then break it down per section. I can remove it though